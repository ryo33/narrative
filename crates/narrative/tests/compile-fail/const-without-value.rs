#[narrative::story("Story with const missing value")]
trait ConstWithoutValueStory {
    const INVALID_CONST: i32; // Error: in a story, all consts must have a value

    #[step("When using const")]
    fn use_const(&self);
}

fn main() {}
