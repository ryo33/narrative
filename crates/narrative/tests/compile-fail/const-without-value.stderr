error: in a story, all consts must have a value
 --> tests/compile-fail/const-without-value.rs:5:5
  |
5 |     #[step("When using const")]
  |     ^
